components/brave_new_tab_ui/async/brave_vpn.ts

```diff
@@ -26,7 +26,9 @@ const observer = {
 
   onPurchasedStateChanged: (state: PurchasedState, description: string) => {
     store.dispatch(Actions.purchasedStateChanged(state))
-  }
+  },
+
+  onSmartProxyRoutingStateChanged: (enabled: boolean) => {}
```

Comment:
Tested this with the NTP refresh and it works great! Just a heads up @zenparsing 😄👍